main.js

```diff
@@ -0,0 +1,22 @@
+let isClicked = false;
+
+function openNav() {
+  if (!isClicked) {
+    document.getElementById('Mynav').style.display = 'block';
+    document.getElementById('head').style.filter = 'blur(0.3rem)';
+    document.getElementById('logo').style.filter = 'blur(0.3rem)';
+    document.getElementById('hamb').style.display = 'none';
+    isClicked = true;
+  } else {
+    document.getElementById('Mynav').style.display = 'none';
+    document.getElementById('head').style.filter = 'none';
+    document.getElementById('hamb').style.display = 'block';
+    document.getElementById('logo').style.filter = 'none';
+    isClicked = false;
+  }
+}
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Kindly make the mobile menu background a bit more blurred. As you can see that the mobile menu options are not clearly visible. kindly fix it according to the Figma design.

- Figma design:

<img width="340" alt="Screenshot 2022-05-10 at 11 59 35 AM" src="https://user-images.githubusercontent.com/70269507/167591239-f63a73df-f3c0-46f8-9bf2-903d79ece681.png">

-Your design:

<img width="435" alt="Screenshot 2022-05-10 at 12 00 02 PM" src="https://user-images.githubusercontent.com/70269507/167591251-9f3efbc2-2ea8-41e3-aa91-27adf9b52cb6.png">